apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -179,6 +179,18 @@ export default class EnrollmentsPanel extends React.Component {
     });
   };
 
+  getViewSurveyUrl = (workshopId, course, subject) => {
+    if (
+      !['CS Discoveries', 'CS Principles', 'CS Fundamentals'].includes(course)
+    ) {
+      return null;
+    }
+
+    return course === 'CS Fundamentals' && subject === 'Intro'
+      ? `/pd/workshop_dashboard/survey_results/${workshopId}`
+      : `/pd/workshop_dashboard/daily_survey_results/${workshopId}`;
+  };
```

Comment:
Does this logic already exist someplace in our Rails application?  If not, I think it's okay to add it here for now, but it'd be nice to consolidate knowledge of routes to the server and pass them down as needed.  Or even better - have only one "workshop survey results" route that worked for all workshop types. :shrug:

Reply:
The generic route is where we're heading toward. Once CSF Intro workshops also use the new survey form, all survey requests will be consolidated to 1 endpoint. Other survey view routes are legacy and can be removed.

For now the generic server endpoint is
https://github.com/code-dot-org/code-dot-org/blob/8e9145bb83e91c94eee810263fe6f6159d0e1a23/dashboard/app/controllers/api/v1/pd/workshop_survey_report_controller.rb#L113-L122
but later it will be in the Foorm system
https://github.com/code-dot-org/code-dot-org/blob/3cce2fef380a99ea02f7872550214982e58620e5/dashboard/app/controllers/api/v1/pd/workshop_survey_foorm_report_controller.rb#L6-L9

Reply:
we will be in a different case for a while where some surveys will be using foorm and others using jotform, which will have different view urls most likely. So we may need to keep this if/else around (with different conditions) for a while.